internal/services/contracts_events_consumer.go

```diff
@@ -34,9 +34,11 @@ type ContractsEventsConsumer struct {
 type EventName string
 
 const (
-	PrivilegeSet                EventName = "PrivilegeSet"
-	AftermarketDeviceNodeMinted EventName = "AftermarketDeviceNodeMinted"
-	Transfer                    EventName = "Transfer"
+	PrivilegeSet                 EventName = "PrivilegeSet"
+	AftermarketDeviceNodeMinted  EventName = "AftermarketDeviceNodeMinted"
+	Transfer                     EventName = "Transfer"
+	BeneficiarySet               EventName = "BeneficiarySet"
+	AftermarketDeviceTransferred EventName = "AftermarketDeviceTransferred"
```

Comment:
The transfer event for aftermarket devices is the standard one, we're already handling it as `Transfer` up above. So I think to handle beneficiary cleanup, we might just want to modify `handleAfterMarketTransferEvent`.